### PR TITLE
website/integrations: Add Gitea Helm Chart Configuration

### PR DESCRIPTION
**Pull request by bpanesar**
# Details
* Adds documentation on how to configure Authentik in Gitea using the Gitea Helm Chart

### New Features
* Adds documentation on how to configure Authentik in Gitea using the Gitea Helm Chart

### CODE REVIEW

File: website/integrations/services/gitea/index.md [hunk 1]
```diff
@@ -69,3 +69,55 @@ Change the following fields
 ![](./gitea1.png)
 
 `Add Authentication Source` and you should be done. Your Gitea login page should now have a `Sign in With` followed by the authentik logo which you can click on to sign-in to Gitea with Authentik creds.
+
+## Helm Chart Configuration
+
+Authentik can be configured automatically in Gitea Kubernetes deployments via it's [Helm Chart](https://gitea.com/gitea/helm-chart/).
```
Comment (BeryJu) [inline website/integrations/services/gitea/index.md:75]: Nit: should be lowercased as stated here https://goauthentik.io/developer-docs/docs/writing-documentation#general-guidelines

File: website/integrations/services/gitea/index.md [hunk 2]
```diff
@@ -69,3 +69,55 @@ Change the following fields
...
+:::note
+This is based on Authentik 2022.8.2, Gitea v17.2, and Gitea Helm Chart v6.0.1. Instructions may differ between versions.
+:::
+
+Add the following to the Gitea Helm Chart `values.yaml` file:
+
+```yaml
+gitea:
+  oauth:
+    - name: 'Authentik'
```
Comment (BeryJu) [inline website/integrations/services/gitea/index.md:86]: Same as above

File: website/integrations/services/gitea/index.md [hunk 3]
```diff
@@ -69,3 +69,55 @@ Change the following fields
 ![](./gitea1.png)
 
 `Add Authentication Source` and you should be done. Your Gitea login page should now have a `Sign in With` followed by the authentik logo which you can click on to sign-in to Gitea with Authentik creds.
+
+## Helm Chart Configuration
+
+Authentik can be configured automatically in Gitea Kubernetes deployments via it's [Helm Chart](https://gitea.com/gitea/helm-chart/).
+
+:::note
+This is based on Authentik 2022.8.2, Gitea v17.2, and Gitea Helm Chart v6.0.1. Instructions may differ between versions.
```
Comment (BeryJu) [inline website/integrations/services/gitea/index.md:78]: Same as above

File: website/integrations/services/gitea/index.md [hunk 4]
```diff
@@ -69,3 +69,55 @@ Change the following fields
...
+  key: 'CLIENT_ID_FROM_AUTHENTIK' #Step 1
+  secret: 'CLIENT_SECRET_FROM_AUTHENTIK' #Step 1
+```
+
+Add the following to the Gitea Helm Chart `values.yaml` file:
+
+```yaml
+gitea:
+  oauth:
+    - name: 'Authentik'
```
Comment (BeryJu) [inline website/integrations/services/gitea/index.md:117]: Same as above

**Review by BeryJu — CHANGES_REQUESTED** [review]
Small nitpicks, needs `make website` to fix formatting

**Comment by bpanesar** [comment]
> Small nitpicks, needs `make website` to fix formatting

Should I squash my

**Pull request merged**